Microsoft.Toolkit.HighPerformance/Microsoft.Toolkit.HighPerformance.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard1.4;netstandard2.0;netstandard2.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
```

Comment:
Does this need anything new for our build pipeline/tooling requirements for folks?

Reply:
Commenting here about the same comments from the stream, for future reference. Right now this works on the latest versions of VS stable (not the preview preview), because it uses the same Roslyn codebase as VS Preview, just from an older branch. As such, some C# 9 features are buggy/outdated (eg. function pointers), but since here we're really only using the new nint type, that bit in particular _mostly_ works. I only had to add an extra explicit cast here to ensure the correct codegen due to a small compiler bug that's fixed in VS Preview, for instance:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/7a65a29ef8c0570c7c66a20cf2bb45d72a3066b2/Microsoft.Toolkit.HighPerformance/Helpers/Internals/SpanHelper.Count.cs#L277-L280

As in, this PR should work fine when compiled with VS stable, but especially due to all the other C# 9 changes coming in other PRs, as well as the .NET 5 support itself, we should definitely update the CI build of VS to the new release as soon as it drops hopefully next week when .NET 5 is announced at .NET Conf 😄